### ISSUE TITLE
Uncensored videotape + stills from HD collection

### ISSUE THREAD

**Issue by snigelpasta** [elishacloud/Silent-Hill-2-Enhancements#141]
As the original version of the videotape is in the files of the PC edition, I would suggest making it possible to have it play with the original sound. I can see why some players would not want this as it's not in the original release of the game, but I personally prefer it by a long shot, because it's what the developers originally wanted and because it's easier to understand what is going on.

In any case, I absolutely think this should be made optional. It would be really easy to make an alternative video file without re-encoding just by replacing the audio track with the uncensored one in Bink.

What do you guys think of this suggestion?

Also, has anyone extracted all the still images from the HD Collection? I think Hijinx had artists who painted the sides of the images to make them fit better in widescreen. Some of these do not really work, but I think for example the one with the map in James' car is wonderful and could absolutely be put in the Enhanced Edition.

**Comment by Polymega** [comment]
Hi @snigelpasta ,

> because it's what the developers originally wanted and because it's easier to understand what is going on.

The developers removed this audio before final release--along with altering other things such as the intensity of Eddie puking--which means its removal was ultimately intended by them. 

On that note, I'll go ahead and say we **cannot** [restore the audio of Mary reading her letter to Laura](https://youtu.be/CquVMeK7Q3g?t=67). This is because the speed in which the letter can be read is entirely controlled by the player; so we can't "force" the letter to be read at a timed speed that syncs with the audio.

> In any case, I absolutely think this should be made optional.

This could certainly be a fun inclusion down the line. I'll go ahead and say that adding such a feature would be something that could be done near the end of this project. There are still a lot of bugs (both small and large) and other generalized improvements we aim to achieve first, which would take priority over adding in bonus content at this time. 

> Also, has anyone extracted all the still images from the HD Collection? 

I believe they've all been extracted. Aero_ and I have looked through them all in the past. While some are fairly well redone for widescreen, many were done poorly. We're going to use AI neural networking to enhance the original fullscreen images and adjust them from there to really make them nice and pretty. [You can follow progress on that by visiting this thread.](https://github.com/elishacloud/Silent-Hill-2-Enhancements/issues/48)

**Comment by snigelpasta** [comment]
> Hi @snigelpasta ,
> 
> because it's what the developers originally wanted and because it's easier to understand what is going on.
> 
> The developers removed this audio before final release--along with altering other things such as the intensity of Eddie puking--which means its removal was ultimately intended by them.
> On that note, I'll go ahead and say we cannot restore the audio of Mary reading her letter to Laura. This is because the speed in which the letter can be read is entirely controlled by the player; so we can't "force" the letter to be read at a timed speed that syncs with the audio.
> 
> In any case, I absolutely think this should be made optional.
> 
> This could certainly be a fun inclusion down the line. I'll go ahead and say that adding such a feature would be something that could be done near the end of this project. There are still a lot of bugs (both small and large) and other generalized improvements we aim to achieve first, which would take priority over adding in bonus content at this time.
> 
> Also, has anyone extracted all the still images from the HD Collection?
> 
> I believe they've all been extracted. Aero_ and I have looked through them all in the past. While some are fairly well redone for widescreen, many were done poorly. We're going to use AI neural networking to enhance the original fullscreen images and adjust them from there to really make them nice and pretty. You can follow progress on that by visiting this thread.

Reading the letter is one thing, but the screaming from the videotape can absolutely be restored as it's part of a pre-rendered video.

Are you sure this was done by Team Silent? I can see why Konami would want it to be less gruesome but not Team Silent; considering it makes it much clearer what is going on. On my first playthrough, I didn't understand that a murder was taking place.

Oh well.

**Comment by Polymega** [comment]
Konami could certainly be to blame for its removal; they don't have the best track record, after all. I could try and ask Masahiro Ito about it later on, to see what he says. 

In any case, I also think it'd be a fun bonus thing to add down the line, once we address the bigger issues/improvements we're needing to do.

**Comment by snigelpasta** [comment]
Fair enough.

And yeah, some confirmation from Ito-san himself would be awesome.

I do agree that most of the HD Collection widescreen images are garbage (especially the clock, lol) but I see no reason not to use the good ones. Is there anywhere I can find these HD Collection images by the way? I could only find the menu images in the thread you linked.

When the time comes to upscale/improve the pre-rendered cutscenes, i'd love to make a version of the videotape that's transferred to an actual VHS tape and then back, for a more convincing look. I'm in the field of post production and would be able to it super well. I saw a processed version of this particular scene on YouTube and especially the VHS fuzz looked awful with this process so I think doing this would be a creative and feasible way to make it look much more authentic and presentable next to HD gameplay.

Also, as I said i'm no expert at this but if someone were to process the first FMV in the bathroom, would it be possible to input some detail information from this promotional photo that I assume was rendered from the same source files but in higher resolution? http://www.silenthillmemories.net/sh2/wallpapers/pics/sh2_15_1024.jpg

Also, one last question.
You know that improved Toluca Lake image that was added into the parking lot a couple months back? Please god tell me that was made and saved in a higher resolution in case someone finds away to put higher resolution textures into the game later down the road.

Best of wishes.

**Comment by Polymega** [comment]
I'm going to close this ticket. We've actually used some of the HD Collection's fullscreen images as a base for when we made our own upscaled, full screen images. We plan to release a new update for the FMV Enhancement Pack that will make them [look a little bit nicer.](https://github.com/elishacloud/Silent-Hill-2-Enhancements/issues/402#issuecomment-805399613)

Looking back, we're probably not going to offer an alternate audio version of hearing Mary suffocating--we'll leave that as a cool thing for people to learn about on fansites or YouTube videos.
